src/components/button/index.js

```diff
@@ -175,8 +175,8 @@ export default class Button extends BaseComponent {
     }
   }
 
-  componentDidUpdate() {
-    if (this.props.animateLayout) {
+  componentDidUpdate(prevProps) {
+    if (!_.isEqual(prevProps, this.props) && this.props.animateLayout) {
```

Comment:
One more thing I just noticed...
Better check `this.props.animateLayout` before checking the equality of the props
since the equal check is more expensive and can be avoided if the `animateLayout` prop is off.

Reply:
done.